cmd/loop/main.go

```diff
@@ -257,13 +261,9 @@ func getOutLimits(amt btcutil.Amount,
 	return &outLimits{
 		maxSwapRoutingFee:   maxSwapRoutingFee,
 		maxPrepayRoutingFee: maxPrepayRoutingFee,
-
-		// Apply a multiplier to the estimated miner fee, to not get
-		// the swap canceled because fees increased in the mean time.
-		maxMinerFee: btcutil.Amount(quote.HtlcSweepFeeSat) * 100,
-
-		maxSwapFee:   btcutil.Amount(quote.SwapFeeSat),
-		maxPrepayAmt: maxPrepayAmt,
+		minMinerFee:         btcutil.Amount(quote.HtlcSweepFeeSat),
+		maxSwapFee:          btcutil.Amount(quote.SwapFeeSat),
+		maxPrepayAmt:        maxPrepayAmt,
```

Comment:
can't we just set `maxMinerFee: maxPrepayAmt` here rather than rely on the field being populated outside of this function?

Reply:
Yep this is much better!